src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -676,6 +676,31 @@ def _target_filtering_info(ctx):
     else:
         return None
 
+
+def _expand_copt(ctx, target, input_copt):
+    """
+    Process make variables in coptss
+    https://docs.bazel.build/versions/master/be/make-variables.html
+    Note: this requires all to be declared as a pre-requisite
+    """
+    if not "$(" in input_copt:
+        return input_copt
+    expanded_copt = ctx.expand_location(input_copt)
+    if "bazel-out/" in expanded_copt:
```

Comment:
Hey @DavidGoldman If you've got a min, I wanted to get your input on this one, I think to make this robust we can't hardcode use `bazel-out` as people _can_ change it. Perhaps we can get it by splitting `ctx.bin_dir` and taking the first part.. https://docs.bazel.build/versions/master/skylark/lib/ctx.html#bin_dir  . We might be able to re-work `_convert_outpath_to_symlink_path` to not strip the leading characters up to that, what do you thnk?